### ISSUE TITLE
Blender crashing with OSM buildings when switching to material preview or Eevee rendering

### ISSUE THREAD

**Issue by tierradenadie** [vvoovv/blender-geo#239]
I have a strange issue with the blender-osm addon. Whenever I try to switch to "Material preview" or "Rendered" with Eevee as render engine in the Viewport Shading settings with imported OSM buildings in the scene, blender crashes completly. As it only happens after importing the buildings, there might be a problem with Eevee and the building materials. Cycles works just fine.

I added the crash log.

Addon version is 2.4.31
Asset version is 2021.01.05

[Test.crash.txt](https://github.com/vvoovv/blender-osm/files/5808122/Test.crash.txt)

**Comment by vvoovv** [comment]
Hi @tierradenadie 

Unfortunately I am unable to reproduce the problem. Nobody else reported a similar problem so far.

What can I suggest:
* reinstall Blender
*  try it in a previous version of Blender

**Comment by tierradenadie** [comment]
Thank you for your quick answer, but the problemed somehow solved itself.

Okay, this is crazy. I didn't want to reinstall Blender completly, so I wouldn't had to reinstall my add-ons. So I downloaded the portal version and fired it up. It had all the same settings and add-ons activated as the installed version, but suddenly the problem did not occur anymore. I removed the portable version and started the installed one and it also works. I don't know exactly what happened, but I'm happy with the result.
